README.md

```diff
@@ -1 +1,8 @@
-# developer-checklist
\ No newline at end of file
+# Developer Checklist
+
+## Open the Perfect PR
+1. Create a new branch `your_short_name/feature-description` (Example: `demi/fixing-duplicate-port-bug`). When making a branch, we use our names to distinguish who does the feature.
+1. The PR itself should also begin with `your_short_name/<PR title>` for similar reasons
+    - PR Title should be as short and concise as possible
+1. PRs submitted during your time at Colonist follow a strict format and criteria, both for the [PR description](/docs/pr-format.md), and code quality
+1. Before submitting a PR, ensure it abides by [Developer fundamentals](/docs/developer/developer-fundamentals.md), [Developer Intermediate](/docs/developer/developer-intermediate.md), and [Developer advanced](/docs/developer/developer-advanced.md)
```

Comment:
developer/developer-...

The repo is also named developer. Remove the unnecessary developers.